include/wsrep/server_state.hpp

```diff
@@ -554,11 +554,7 @@ namespace wsrep
         }
 
         enum state state(wsrep::unique_lock<wsrep::mutex>&
-                         lock WSREP_UNUSED) const
-        {
-            assert(lock.owns_lock());
-            return state_;
-        }
+                         lock WSREP_UNUSED) const;
```

Comment:
I think this WSREP_UNUSED is no longer necessary.
Not sure, perhaps we want to remove it.